multiple_generate.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+while getopts s:e:u: option
+do
+ case "${option}"
+ in
+ s) SUBPROJECT=${OPTARG};;
+ e) EMAIL=${OPTARG};;
+ u) UNIQUEID=${OPTARG};;
+ esac
+done
+
+BASE=$(pwd)
+
+if [ "${WEBUI:-false}" == "true" ]; then
+  BUILD_DIR=$(pwd)/temp/$EMAIL/$UNIQUEID
+  mkdir -p $BUILD_DIR
+  cd $BUILD_DIR
+else
+  git clone -q https://github.com/fossasia/yaydoc.git yaydocclone
+  BASE=$(pwd)/yaydocclone
+  BUILD_DIR=$(pwd)
+fi
+
+
+IFS=', ' read -r -a SUBPROJECTS <<< "$SUBPROJECT"
+
+for element in "${SUBPROJECTS[@]}"
+do
+  URL_SPLIT=(${element//// })
+  REPONAME=(${URL_SPLIT[3]//./ })
+  git clone $element ${REPONAME}_temp &
+done
+
+wait
+
+for element in "${SUBPROJECTS[@]}"
+do
+  URL_SPLIT=(${element//// })
+  REPONAME=(${URL_SPLIT[3]//./ })
+  cd ${REPONAME}_temp
+  mkdir ../$REPONAME
+  cp -a docs/. ../$REPONAME/.
+  cd ..
+  sudo rm -r ${REPONAME}_temp
+done
+
+cp -a ${BASE}/modules/scripts/ $BUILD_DIR/
+cp -a ${BASE}/modules/templates/ $BUILD_DIR/
+
+mkdir _themes
+
+cp -a ${BASE}/fossasia_theme $BUILD_DIR/_themes/
+
+URL_SPLIT=(${SUBPROJECTS[0]//// })
+USERNAME=${URL_SPLIT[2]}
+REPONAME=(${URL_SPLIT[3]//./ })
+pip install -q -r $BASE/modules/requirements.txt
+sphinx-quickstart -q -v "($(date +%Y-%m-%d.%H:%M:%S))" -a $USERNAME -p $REPONAME -t templates/ -d html_theme=alabaster -d html_logo=$LOGO
+
+rm index.rst
+
+python $BASE/modules/scripts/genindex.py ./
```

Comment:
One last issue, what about the README's of the sub-projects. since they won't be in subproject/docs, they won't be included in the build. How about, if we copy the entire repo inside BUILD_DIR and visit subproject/docs and generate index if needed for each subproject and then, for the main index, we just add a toctree with links to all sub-indices? views @sch00lb0y @imujjwal96